x-pack/plugins/apm/public/components/app/service_map/Popover/stats_list.tsx

```diff
@@ -82,6 +87,8 @@ export function StatsList({ data, isLoading }: StatsListProps) {
           ? asDuration(transactionStats?.latency?.value)
           : null,
```

Comment:
Unrelated but can you remove this ternary. `asDuration` already handles the `null` case